cli/enry/main.go

```diff
@@ -136,6 +139,50 @@ func printPercents(out map[string][]string, buff *bytes.Buffer) {
 	}
 }
 
+func printFile(file string, buff *bytes.Buffer) {
+	content, err := ioutil.ReadFile(file)
+	if err != nil {
+		fmt.Println(err)
+	}
+	totalLines, sloc := getLines(file, content)
+	fileType := getFileType(file, content)
+	language := enry.GetLanguage(file, content)
+	mimeType := enry.GetMimeType(language)
+
+	buff.WriteString(fmt.Sprintf("%s: %d lines (%d sloc)\n", filepath.Base(file), totalLines, sloc))
+	buff.WriteString(fmt.Sprintf("  type:      %s\n", fileType))
+	buff.WriteString(fmt.Sprintf("  mime type: %s\n", mimeType))
+	buff.WriteString(fmt.Sprintf("  language:  %s\n", language))
+}
+
+func getLines(file string, content []byte) (int, int) {
+
+	totalLines := bytes.Count(content, []byte("\n"))
+	sloc := totalLines - strings.Count(string(content), "\n\n")
+
+	return totalLines, sloc
+}
+
+func getFileType(file string, content []byte) string {
+	switch {
+	case isImage(file):
+		return "Image"
+	case enry.IsBinary(content):
+		return "Binary"
+	default:
+		return "Text"
+	}
+}
+
+func isImage(file string) bool {
+	index := strings.LastIndex(file, ".")
+	extension := file[index:]
+	if extension == ".png" || extension == ".jpg" || extension == ".jpeg" || extension == ".gif" {
```

Comment:
the isImage func was directly translated from linguist, those are the format that they support